### ISSUE TITLE
[Bug] v2.2.0 Crash on start after upgrade

### ISSUE THREAD

**Issue by sfc9982** [Zettlr/Zettlr#3180]
### Description

Zettlr crashes on start after install the 2.2.0.

Tried:
  - Re-install
  - Clear the cache in appdata
  - Re-start Windows

The log file only shows:
`[Info] Migrating from 2.1.3 to 2.2.0!`

### Reproducing

Maybe do a same upgrade

### Zettlr Version

Stable (most recent version)

### Specify version

2.2.0

### Your Platform

- [X] Windows
- [ ] macOS
- [ ] Linux

### Architecture

- [X] Intel 64bit
- [ ] ARM 64bit

### Operating System Version

Windows 11 21H2

**Comment by carlmartus** [comment]
Zettlr crashes for me as well.

I could run version `2.2.0` after i upgraded from `2.1.3` for a day. But the next day I couldn't start Zettlr anymore.
If I run `Zettlr` in a terminal I get the following output in the terminal:

```
Performing environment check ...
[Application] Pandoc has been bundled with this release. Path: /opt/Zettlr/resources/pandoc
Environment check complete.
[4227:0212/083720.123490:ERROR:sandbox_linux.cc(376)] InitializeSandbox() called with multiple threads in process gpu-process.
RangeError: Maximum call stack size exceeded
    at String.includes (<anonymous>)
    at e (/opt/Zettlr/resources/app.asar/.webpack/main/index.js:2:1632193)
    at e (/opt/Zettlr/resources/app.asar/.webpack/main/index.js:2:1632458)
    at e (/opt/Zettlr/resources/app.asar/.webpack/main/index.js:2:1632458)
    at e (/opt/Zettlr/resources/app.asar/.webpack/main/index.js:2:1632458)
    at e (/opt/Zettlr/resources/app.asar/.webpack/main/index.js:2:1632458)
    at e (/opt/Zettlr/resources/app.asar/.webpack/main/index.js:2:1632458)
    at e (/opt/Zettlr/resources/app.asar/.webpack/main/index.js:2:1632458)
    at e (/opt/Zettlr/resources/app.asar/.webpack/main/index.js:2:1632458)
    at e (/opt/Zettlr/resources/app.asar/.webpack/main/index.js:2:1632458)
```

Let me know if I should upload any more info from my side.

**Comment by hildebrand-institute** [comment]
I have the same exact problem here on Linux Garuda (Arch) with Zettlr AppImage file. Yesterday I could work the whole day long with no issues, but now the AppImage file does not activate anymore...

**Comment by aamatu** [comment]
Same problem here on mac os with the arm64 version.

**Comment by nathanlesage** [comment]
Patch is just now building, so it'll be available in 30 minutes or something like that!

**Comment by nathanlesage** [comment]
Fixed with 2.2.1, released a few minutes ago.
